ui/page/privacy/account_mixer_page.go

```diff
@@ -29,8 +27,9 @@ type AccountMixerPage struct {
 	// helper methods for accessing the PageNavigator that displayed this page
 	// and the root WindowNavigator.
 	*app.GenericPageModal
-	*listeners.AccountMixerNotificationListener
-	*listeners.TxAndBlockNotificationListener
+
+	accountMixerNotificationListener *dcr.AccountMixerNotificationListener
+	txAndBlockNotificationListener   *sharedW.TxAndBlockNotificationListener
 
```

Comment:
With the pattern adopted in this PR, I think these should also go. We don't need to keep track of them.